Dockerfile

```diff
@@ -86,4 +96,4 @@ RUN casmsmartphase generate-bed --version
 
 WORKDIR /home/ubuntu
 
-CMD ["/bin/bash"]
+CMD ["java","-jar", "/opt/wsi-t78/smartPhase.jar"]
```

Comment:
To be clear, this makes it less obvious that the container can be used for more than just the jar file execution as running without a command will only run smartPhase rather than give a prompt.

Reply:
Fair point. Reverted and changed README